example/bwmf/block_parameter.go

```diff
@@ -0,0 +1,137 @@
+package bwmf
+
+import (
+	"fmt"
+
+	pb "github.com/taskgraph/taskgraph/example/bwmf/proto"
+	op "github.com/taskgraph/taskgraph/op"
```

Comment:
then we have to use taskgraph_op as the pkg name and taskgraph is somehow redundant.